deno/rest/v10/monetization.ts

```diff
@@ -0,0 +1,91 @@
+import type { Snowflake } from '../../globals.ts';
+import type { APIEntitlement, APISKU } from '../../v10.ts';
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#list-entitlements
+ */
+export interface RESTListAPIEntitlementsQuery {
```

Comment:
Please make sure your types adhere to the naming convention: https://discord-api-types.dev/docs/introduction_to_discord-api-types#project-structure